assets/js/atomic/blocks/product-elements/title/block.tsx

```diff
@@ -120,7 +124,8 @@ export const Block = ( props: Props ): JSX.Element => {
 				disabled={ ! showProductLink }
 				name={ product.name }
 				permalink={ product.permalink }
-				rel={ showProductLink ? 'nofollow' : '' }
+				rel={ 'nofollow' + ( rel && ' ' + rel ) }
```

Comment:
I'd build the second part of this string separately and then assign it here. Also, did you know about [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)? They are cooler than ol' fashioned string concatenation! 😎

Reply:
I've extracted this into a `const`, also changed slightly the condition since the result was sometimes `nofollowundefined` 😅